papaparse.js

```diff
@@ -871,101 +871,104 @@ if (!Array.isArray)
 
 
 	function DuplexStreamStreamer(_config) {
```

Comment:
Does it really make sense to define a function with no code?

Reply:
Yeah I don't really like it either, this was just a requirement of how the uglify conditional compilation works (can't have a function that's not at the top level).
I'll play around with it a little more and see if I can come up with something that makes more sense. Maybe just a console.warn that says that streams aren't supported on browser.